www/css/button.css

```diff
@@ -31,30 +33,118 @@
 update all the yellow buttons on the site
 **/
 .yellow {
-  background-color: #F9D856;
+  background-color: var(--brand-yellow);
 }
 
 .blue {
-  background-color: #3881E5;
+  background-color: var(--brand-blue);
   color: #fff;
 }
 
 /**
 TODO update the NEW buttons on signup to this green
 **/
 .green {
-  background-color: #80BC06;
+  background-color: var(--brand-green);
 }
 
 .red {
-  background-color: #d02424;
+  background-color: var(--brand-red);
   color: #fff;
 }
 
 .purple {
-  background-color: #6b25cf
+  background-color: var(--brand-purple);
 }
 
 .grey-outline {
-  border: 1px solid rgb(204, 204, 204)
+  border: var(--grey-outline)
+}
+
+/**
+ * ==============================================
+ * Dot Pulse
+ * ==============================================
+ */
+.dot-pulse {
```

Comment:
this is an animation I'm adding to buttons when a form is submitted so people know to chill...

Reply:
❤️ this